apps/web/pages/api/documents/[id]/copied.ts

```diff
@@ -0,0 +1,37 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { setCopiedField } from "@documenso/lib/copied";
+import { defaultHandler, defaultResponder, getUserFromToken } from "@documenso/lib/server";
+import prisma from "@documenso/prisma";
+
+async function getHandler(req: NextApiRequest, res: NextApiResponse) {
+  const user = await getUserFromToken(req, res);
+  const { id: recipientId } = req.query;
+
+  if (!user) return;
+
+  if (!recipientId) {
+    res.status(400).send("Missing parameter recipientId.");
+    return;
+  }
+
+  const recipients = await prisma.recipient.findMany({
+    where: {
+      id: +recipientId,
+    },
+  });
+
+  console.log("api/documenso/id/copied", recipients);
+
+  if (!recipients.length) return res.status(200).send(recipients.length);
+
+  recipients.forEach(async (recipient) => {
```

Comment:
This won't work the way you're expecting, you could do a `for of` loop but you'd still need to consider how you would rollback when a later item errors (think transactions)